views/partials/login.ejs

```diff
@@ -13,5 +13,5 @@
 </form>
 
 <div class="container">
-  <button id="signup-action" class="btn btn-warning">Sign up</button>
+  <button id="logout-action" class="btn btn-warning">Sign up</button>
```

Comment:
lol, very good point, they are all from Peking University anyway. Let's keep it English for now.